volatility3/framework/plugins/windows/cmdscan.py

```diff
@@ -67,6 +67,7 @@ def get_filtered_vads(
 
         Args:
             conhost_proc: the process object for conhost.exe
+            size_filter: filter (keep) vads less than this size (bytes)
```

Comment:
I understand the reason for reminding people how `filter` works in the volatility world (filter down to, not filter out), but `(keep)` doesn't seem like the clearest way of doing this.  This should work the way that [python's `filter` function](https://docs.python.org/3/library/functions.html#filter) works...

Perhaps we could go with `size above which vads will not be returned`?